Sources/NationalWeatherService/services/calls/NWS+Forecast.swift

```diff
@@ -13,7 +13,7 @@ extension NationalWeatherService {
     fileprivate func loadForecast(at url: URL, then handler: @escaping ForecastHandler) {
         var components = URLComponents(url: url, resolvingAgainstBaseURL: false)!
         components.queryItems = [
-            URLQueryItem(name: "units", value: "si")
```

Comment:
why was this removed?

Reply:
I did not need the results to be in metric units. I can remove it from the PR if you'd like. The best thing might be to have some kind of config mech for it, but I haven't thought that hard on it yet.

Reply:
Since the library is providing temperature as a [Measurement](https://github.com/WeatherProvider/NationalWeatherService-Swift/blob/562a235eb991d78ad07b0dd473b4e147f889c469/Sources/NationalWeatherService/models/Forecast/ForecastPeriod.swift#L23), the programmer can convert between customary/SI as needed [[Apple Docs](https://developer.apple.com/documentation/foundation/measurement/2799413-converted)]. Just to make sure that NWS is returning SI, I would rather keep the `units=si` in the URL. If you have any other feedback, we can continue to discuss, otherwise let's remove this change.

Reply:
restored to orig